docs/source/migrating-from-gateway.mdx

```diff
@@ -6,20 +6,86 @@ description: An overview of the considerations to make when migrating from Apoll
 
 import { Link } from 'gatsby';
 
+The Apollo Router is a high-performance alternative to the Apollo Gateway, written in the Rust programming language.  Apollo Router offers the same general functionality and feature set as the Apollo Gateway, though the technique in which it is configured is different.
+
+The standard Apollo Router packaging is a static binary that is passed its configuration file at start time, rather than a JavaScript application used as a library.  Its configuration file is hot-reloaded, making it possible to change the behavior of a running server with no downtime.  Rather than writing configuration as JavaScript or TypeScript code, the Apollo Router's configuration is provided as a YAML file and supplied to the Router using an argument.
+
+It is possible to use the Apollo Router as a library, but the Apollo Router aims to make it less necessary to write custom user-code in the Gateway and instead integrate the most common critical features directly using declarative methods.
+
+## Taking inventory of Gateway configuration
+
+Apollo Gateway runs on top of Apollo Server, and both have various surfaces on which configurations can be made.  To migrate from the Apollo Gateway, you'll need to take account of these configuration and customizations, and implement the same behavior using different mechanisms provided by the Apollo Router.  For example, you may be customizing the HTTP headers that your subgraphs receive from the client, or passing specific headers back to the client from specific subgraphs.
+
+It'll be good to make a check-list, and you'll want to start by looking for configuration and customizations these places:
+
+- Environment variables
+- Non-Apollo telemetry and instrumentation (i.e., OpenTelemetry, Datadog)
+- Constructor options passed to `new ApolloGateway({ ... })`
+- Constructor options passed to `new ApolloServer({ ... })` (TODO — is this one important? Is plugins most important?)
+- Specific `plugins` passed to `new ApolloServer({ plugins: [ ... ] })`
+- Custom middleware (e.g., Express, Koa, Fastify)
+
+The sections below provide more details on what to look for in each of these categories.
+
+### Environment variables
+
+Environment variables have implicit effect on some Apollo features, like Managed Federation and usage reporting.  Take note of any `APOLLO_` prefixed environment variables that are being set when starting the host runtime you run your Apollo Gateway and Apollo Server on.
+
+All official Apollo-related environment variables are prefixed with `APOLLO_`.  Common cases are:
+
+- `APOLLO_KEY`
+- `APOLLO_GRAPH_REF`
```

Comment:
Worth noting that deprecated environment variables aren't supported in the router? I can't remember the name, but wasn't there a precursor to APOLLO_GRAPH_REF?